### PR TITLE
LEGACY: Improved Sprint module and added KeyPearl module

### PR DESCRIPTION
**Pull request by FDPUser443**
There are some changes:

Sprint:
 - Added allDirectionsLimitSpeed option which allows to configure movement speed of allDirection;
 - Added jumpDirections option. This one can be useful to move faster while sprinting and jumping backwards;
 - Added usingItem option;

KeyPearl module allows players to use ender pearls without taking them in hand. A player can just press a certain key for this. It also supports mouse buttons. (Sometimes, using a mouse button for this can be even more comfortable and faster than using a key from keyboard)

### CODE REVIEW

**Comment by gabrielvicenteYT** [comment]
Add NoPacket to Sprint as well, so that it can bypass anti-cheats such as Verus and Vulcan, and perhaps add a smart NoPacket option :)

File: src/main/java/net/ccbluex/liquidbounce/features/module/modules/player/KeyPearl.kt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        sendPackets(
+            C09PacketHeldItemChange(pearlInHotbar - 36),
+            C08PacketPlayerBlockPlacement(mc.thePlayer.heldItem),
+            C09PacketHeldItemChange(mc.thePlayer.inventory.currentItem)
+        )
+    }
+
+
+    @EventTarget
+    fun onRender3D(event: Render3DEvent) {
```
Comment (1zun4) [inline src/main/java/net/ccbluex/liquidbounce/features/module/modules/player/KeyPearl.kt]: Please use tick-based events.

Comment (FDPUser443) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/player/KeyPearl.kt]: Ok, I will use it

**Comment by FDPUser443** [comment]
> Add NoPacket to Sprint as well, so that it can bypass anti-cheats such as Verus and Vulcan, and perhaps add a smart NoPacket option :)

What is a smart NoPacket option supposed to do? I'm also not 100% sure how NoPacket option works so I won't add it, sorry. But you can try to add it by creating other pull request :)

**Pull request merged**